### PR TITLE
adds error handling

### PR DESCRIPTION
**Pull request by alexcupit**

### CODE REVIEW

File: src/components/Article.js [hunk 1]
```diff
@@ -1,29 +1,39 @@
...
-    getArticleById(article_id).then((article) => {
-      setArticle(article);
-      setArticleLoading(false);
-    });
-  }, []);
-
-  const { title, author, created_at, body, topic, votes } = article;
+    getArticleById(article_id)
+      .then((article) => {
+        console.log(article, '<prom art');
```
Comment (callumu21) [inline src/components/Article.js]: Console.log can be deleted

File: src/components/ErrorPage.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    } else if (customMsg === 'input uses invalid data type')
+      errDesc =
+        "Hmm, something's not right. Please make sure IDs are entered as numbers";
+  } else {
+    errDesc = "Sorry, we can't find that page";
+  }
+
+  return (
+    <div className='error-container'>
+      <h1>We've found a bump!</h1>
```
Comment (callumu21) [inline src/components/ErrorPage.js:21]: Nice.

**Review by callumu21 — APPROVED** [review]
✅ Animated 404 page
✅ Form validation
✅ Mr. Bump giving me life

Just remove the errant console.log, and this is good to merge 👍

File: src/components/ErrorPage.js (hunk 2 above)
Comment (alexcupit) [reply src/components/ErrorPage.js:21]: I think that line is what I'm most proud of this week.

**Pull request merged**